command/connect/redirecttraffic/redirect_traffic.go

```diff
@@ -62,6 +63,8 @@ func (c *cmd) init() {
 		"Outbound CIDR to exclude from traffic redirection. May be provided multiple times.")
 	c.flags.Var((*flags.AppendSliceValue)(&c.excludeUIDs), "exclude-uid",
 		"Additional user ID to exclude from traffic redirection. May be provided multiple times.")
+	c.flags.StringVar(&c.netNS, "netns", "", "The network namespace where traffic redirection rules should apply."+
+		"This must be a path to the network namespace, e.g. /var/run/netns/foo.")
```

Comment:
When would the network namespace in linux not be located in `/var/run`? Wondering if we could assume the path in the executor for simplicity.

It seems that when running commands like `ip netns exec foo <command>` you don't need to specify the path to the file.

Reply:
Namespaces created from `ip netns` will have an entry under `/var/run/netns`, but namespaces created via other means are not guaranteed to be listed here. See https://bunn.cc/2017/nsenter_network_namespace/ for a better explanation and alternate solution for execing into a namespace.

Reply:
What Blake said! For example, on Kube, it'll live under `/run/netns` as well as under `/var/run/netns`. I think we'd need to support a different path because also that's what the [CNI](https://github.com/containernetworking/cni/blob/master/SPEC.md#attachment-parameters) (container networking interface) does.